### PR TITLE
Added new bundle descriptions

### PR DESCRIPTION
**Pull request by w3scout**

### CODE REVIEW

File: meta/w3scout/contao-compareimagesslider-bundle/en.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+en:
+    title: Image Compare Slider
+    description: >
+        The module provides a modern before-after image slider that allows users to easily and intuitively compare two images. This feature makes it possible to directly visualize the differences between the two images by moving a slider. Ideal for applications in image editing, design, and documenting changes.
+    keywords:
+        - contao
+        - bundle
+        - compere images
```
Comment (xchs) [inline meta/w3scout/contao-compareimagesslider-bundle/en.yml:8]:
```suggestion
        - compare images
```

**Comment by xchs** [comment]
There are also some lint errors: https://github.com/contao/package-metadata/actions/runs/9455313447/job/26044821564?pr=567

**Comment by w3scout** [comment]
> There are also some lint errors: https://github.com/contao/package-metadata/actions/runs/9455313447/job/26044821564?pr=567

I´m sorry. But what exactly is being criticized here? The choice of words?

**Comment by fritzmg** [comment]
No, the liner does not understand combined German nouns. You will need to add them to the German allow list.

Als you will need to use the English names you used in the German translation to the German allow list.

**Review by xchs — APPROVED** [review]

**Pull request merged**